pkg/state/state_nv_peer.go

```diff
@@ -43,15 +43,9 @@ type stateNVPeer struct {
 	stateSkel
 }
 
-type nvPeerRuntimeSpec struct {
-	CPUArch    string
-	OSNameFull string
-	Namespace  string
-}
-
 type nvPeerManifestRenderData struct {
 	CrSpec      *mellanoxv1alpha1.NVPeerDriverSpec
-	RuntimeSpec *nvPeerRuntimeSpec
+	RuntimeSpec *nodeInfoRuntimeSpec
```

Comment:
how is this change related to the cleanup?

Reply:
it is refactoring for common attributes, you want it to be in a separated commit or PR?

Reply:
i prefer to keep it as is was for now.